### PR TITLE
Convert fabric-expandable-video-v2 to ES6

### PR DESCRIPTION
**Pull request by janua**
:clap:

The main changes are:

From
```javascript
if( adSlot.parentNode.classList.contains('top-banner-ad-container') ) {
```
to
```javascript
if (
  adSlot.parentNode &&
  ((adSlot.parentNode: any): Element).classList.contains(
    'top-banner-ad-container'
)
```

From
```javascript
id: 'fabric-expandable-' + (Math.random() * 10000 | 0).toString(16), 
```
to
```javascript
id: `fabric-expandable-${Math.round(Math.random() * 10000).toString(16)}`, 
```

From
```javascript
function open(open) { 
```
to
```javascript
const open = (isOpen) => { 
```

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            return true;
+        });
+    };
+
+    return Object.freeze({
+        create,
+    });
+};
+
+export default FabricExpandableVideoV2;
```
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js]: Prefer named exports over default?

Line 18 could be `export const FabricExpandableVideoV2`, then we don'r need a `default` here.. YMMV 😄

Comment (janua) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js]: @katebee Yeah I have been avoiding changing the default exports in the first PR as not to clash with other peoples changes in files...

Comment (janua) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js]: All references already ES6. 😄

**Review by katebee — APPROVED** [review]
LGTM 💎 👍

**Pull request merged**